test/driver_test.rb

```diff
@@ -131,6 +130,73 @@
   end
 
   describe "total_revenue" do
-    # You add tests for the total_revenue method
+    before do
+      @driver = RideShare::Driver.new(
+        id: 54,
+        name: "Rogers Bartell IV",
+        vin: "1C9EVBRM0YBC564DZ"
+      )
+      trip1 = RideShare::Trip.new(
+        id: 8,
+        driver: @driver,
+        passenger_id: 3,
+        start_time: Time.new(2016, 8, 8),
+        end_time: Time.new(2016, 8, 8),
+        cost: 10,
+        rating: 5
+      )
+      @driver.add_trip(trip1)
+
+      trip2 = RideShare::Trip.new(
+        id: 7,
+        driver: @driver,
+        passenger_id: 3,
+        start_time: Time.new(2016, 8, 8),
+        end_time: Time.new(2016, 8, 8),
+        cost: 5,
+        rating: 5
+      )
+      @driver.add_trip(trip2)
+      
+    end
+
+    it "correctly calculates total_revenue" do
+      expect(@driver.total_revenue).must_be_close_to ((5-1.65) + (10-1.65)) * 0.80, 0.01
+    end
+
+    it "returns a float" do
+      expect(@driver.total_revenue).must_be_kind_of Float
+    end
+  end
+
+  describe "total_revenue" do
+    it "If trip cost is less or equal to 1.65 to return 0" do
+      @driver = RideShare::Driver.new(
+          id: 54,
+          name: "Rogers Bartell IV",
+          vin: "1C9EVBRM0YBC564DZ"
+        )
+      trip1 = RideShare::Trip.new(
+        id: 8,
+        driver: @driver,
+        passenger_id: 3,
+        start_time: Time.new(2016, 8, 8),
+        end_time: Time.new(2016, 8, 8),
+        cost: 1.50,
+        rating: 5
+      )
+      @driver.add_trip(trip1)
+      expect(@driver.total_revenue).must_equal 0 
+    end
+
+    it "returns zero if no driven trips" do
+      driver = RideShare::Driver.new(
+        id: 54,
+        name: "Rogers Bartell IV",
+        vin: "1C9EVBRM0YBC564DZ"
+      )
+      expect(driver.total_revenue).must_equal 0
+      expect(driver.total_revenue).must_be_kind_of Numeric
```

Comment:
You don't need to test that `total_revenue` is both `Numeric` and equal to `0` since `0` is itself `Numeric`.

```suggestion
      expect(driver.total_revenue).must_equal 0
```